CMakeLists.txt

```diff
@@ -23,28 +26,41 @@ SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
 # einfach den code mit magic macros vollscheissen, bis es geht :o)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_POSIX_C_SOURCE -D_BSD_SOURCE -D_DEFAULT_SOURCE -D_GNU_SOURCE")
 
+# use null sink otherwise
+option(ENABLE_LOGGING "whether logging should be enabled" ON)
+if (ENABLE_LOGGING)
+    target_compile_definitions(roco2 INTERFACE ENABLE_LOGGING)
+else()
+    message(STATUS "silencing all logging")
+endif()
+
 find_package(OpenMP REQUIRED)
 target_link_libraries(roco2 INTERFACE OpenMP::OpenMP_CXX)
 
 # this should also find MKL
 find_package(BLAS REQUIRED)
-target_link_libraries(roco2 INTERFACE BLAS::BLAS)
+target_link_libraries(roco2 INTERFACE ${BLAS_LIBRARIES})
```

Comment:
I think I'd rather increase the required CMake version to 3.18.

Reply:
I'd argue against increasing the required cmake version:

1. this is the only part in the code where a higher cmake version would be required
2. research systems are riddled with old releases (including our local testing systems, a recompilation of at least 2 libs would be required) -- why break compatibility (jumping 7 minor versions!) for an optical change